src/framework/preferences/js/SeparatedPanelPrefsEditor.js

```diff
@@ -292,6 +292,9 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         }, "collapseFrame");
         separatedPanel.slidingPanel.events.afterPanelShow.addListener(function () {
             separatedPanel.iframeRenderer.iframe.show();
+
+            // FLUID-6183: Required for bug in MS EDGE that clips off the bottom of adjusters
```

Comment:
Could you please describe in the comment why it is that the code below fixes the issue in MS Edge.